docs/javascripts/downloads.js

```diff
@@ -0,0 +1,176 @@
+(function() {
+  const downloadsTableIdentifier = document.getElementById('downloads-table-identifier');
+
+  if (downloadsTableIdentifier) {
```

Comment:
I'd prefer to invert this to reduce the huge indented code block below, i.e. `if (!downloadsTableIdentifier) { return; }`